### PR TITLE
Creation of the Games folder and adding first game

### PR DESCRIPTION
**Pull request by nuggreat**
As discussed in the slack this is the creation of the games folder and adding the first game to it.

### CODE REVIEW

File: games/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## KS-lib/games
+
+This is a public repository for games written in kerboscript
+
+### basic layout
```
Comment (scimas) [inline games/README.md]:
```suggestion
### Basic Layout
```

File: games/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Each has it's own sub folder for ease of organization.
```
Comment (scimas) [inline games/README.md]:
```suggestion
Each game has its own sub folder for ease of organization.
```

File: games/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+With in each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game
```
Comment (scimas) [inline games/README.md]:
```suggestion
Within each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game.
```

File: games/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+Additionally there may also be needed libraries for the game in that case they can ether be found in the game folder or linked to in the README file.
```
Comment (scimas) [inline games/README.md]:
```suggestion
Additionally there may also be needed libraries for the game, which can either be placed in the game folder or linked to in the README file.
```

File: games/SUMMARY.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Summary
+
+This is a summary of all games found in KS-lib
```
Comment (scimas) [inline games/SUMMARY.md]:
```suggestion
This is a summary of all games found in KSLib.
```

File: games/SUMMARY.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+### Game of Ur
+
+  This is a kOS version of the board game "The Royal Game Of Ur". 
+    This is a 2 player race game where the first person to get all there pieces off the board is the winner.
```
Comment (scimas) [inline games/SUMMARY.md]:
```suggestion
  This is a 2 player race game in which the first person to get all of their pieces off the board is the winner.
```

File: games/SUMMARY.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+	Too allow for a single person to play there are several AI.
```
Comment (scimas) [inline games/SUMMARY.md]:
```suggestion
	To allow for a single person to play, several AIs are offered.
```

File: HOWTO.md [hunk 8]
```diff
@@ -35,6 +35,8 @@ general patterns:
   * doc/
     * A place for you to describe your library and how to use it, in
       textual form, using ascii text, or Github markdown files (.md).
+  * games/
+    * A place to find kOS games made for kOS, more detailed information can be found in the README.md file with in
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply HOWTO.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
+# Game Of Ur
+
+The Game of Ur is a game played in ancient Mesopotamia.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+As there are no dependencies for this script you can simply copy it into your archive and run it.
+
+## Rules
+
+This is a two player game, one player is 'X' the other is 'O'.
+There are several AI options should you wish to play by your self.
+To play you roll dice to determine how far you move in a turn.
+The way to Win is to get all 7 of your pieces off the board.
+A player can send an opponents pieces back to the start by landing on them with there own piece.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+There are 5 areas on the board and players may only interact directly in the Main area.
+ 
+At the start of your turn you roll four sided dice.
+Each die has 2 white and 2 black dots with one dot on each face.
+The number of white dots are added together to get the number of spaces you will move.
+As you can get no white on your roll it is possible to end up skipping your turn.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+There are 5 tile that when landed on allow the player to roll again
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/README.md (hunk 3 above)
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/SUMMARY.md (hunk 5 above)
Comment (nuggreat) [reply games/SUMMARY.md]: made recommended change

File: games/SUMMARY.md (hunk 6 above)
Comment (nuggreat) [reply games/SUMMARY.md]: made recommended change

File: games/SUMMARY.md (hunk 7 above)
Comment (nuggreat) [reply games/SUMMARY.md]: made recommended change

File: games/README.md (hunk 4 above)
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+The probability spread for all possible rolls are:  0 = 1/16, 1 = 2/8 2 = 3/8, 3 = 2/8, 4 = 1/16
+ 
+To get a piece off the board you must roll The exact number.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+Getting off the board must be done with an exact roll.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: removed 2nd sentence

File: games/Game_Of_Ur/README.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+For instance if you are on the last tile then you must roll a 1.
+ 
+A player can only interact with there opponents pieces in the main area of the board.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+To send an opponents piece back to the start a player must simply land on the opponents piece.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: rephrased to "To send an opponent's piece back to the start a player must simply land on the piece they wish to send back."
and added 2nd line of "A player can't land on there own pieces."

File: games/Game_Of_Ur/README.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+The roll again tile in the main area is a save tile and a piece on the tile can't be sent back to the start.
+
+The 5 areas on the board are xStart, oStart, Main, xEnd, oEnd.
+The tiles are numbered from -4 to 9 along the path of movement.
+Some tiles have duplicate numbers because each player has there own start and end areas.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/Game_Of_Ur/README.md [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+There are 2 hidden tiles for starting and ending places.
+The starting area is numbered -5.
+The ending area is numbered 10.
+
+Several Examples of various rules as well as a copy can also be found in the help menu with in the game it's self.
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/Game_Of_Ur/README.md]: made recommended change

File: games/README.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
+## KS-lib/games
+
+This is a public repository for games written in kerboscript
```
[the remark this answers is not in the document]
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/README.md (hunk 1 above)
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/README.md (hunk 2 above)
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/README.md [hunk 20]
```diff
@@ -1,12 +1,12 @@
...
 
-### basic layout
+### Basic layout
 
-Each has it's own sub folder for ease of organization.
-With in each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game
-Additionally there may also be needed libraries for the game in that case they can ether be found in the game folder or linked to in the README file.
+Each game has its own sub folder for ease of organization.
+Within each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game.
+Additionally there may also be needed libraries for the game, which can either ether be placed in the game folder or linked to in the README file for the game.
```
Comment (scimas) [inline games/README.md]: either - ether

File: games/SUMMARY.md [hunk 21]
```diff
@@ -0,0 +1,11 @@
+# Summary
+
+This is a summary of all games found in KSlib
+
+### Game of Ur
+
+  This is a kOS version of the board game "The Royal Game Of Ur". 
+  This is a 2 player race game in which the first person to get all of there pieces off the board is the winner.
```
Comment (scimas) [inline games/SUMMARY.md]:
```suggestion
  This is a 2 player race game in which the first person to get all of their pieces off the board is the winner.
```

File: games/README.md (hunk 20 above)
Comment (nuggreat) [reply games/README.md]: made recommended change

File: games/SUMMARY.md (hunk 21 above)
Comment (nuggreat) [reply games/SUMMARY.md]: made recommended change

**Comment by nuggreat** [comment]
That should be it I hope, thanks for the reviewing of all of this.

**Comment by nuggreat** [comment]
That should do for the last issue you raised.  I changed the `games/README.md` as it was never my intent to have all the game files by called `games.ks` the games part of that was intended as a "your game name here" type thing.  Because if it is was that way then a person having more than one game would be problematic as they would all have the same name.

**Pull request merged**